### PR TITLE
clarify sense of required Validator

### PR DESCRIPTION
**Pull request by infabo**
The current README is annoying.
The required Validator is described like I could "make sure a variable is defined in the dot-env file".
This not the case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,8 +138,8 @@ $dotenv->overload();
 Requiring Variables to be Set
 -----------------------------
 
-Using Dotenv, you can require specific ENV vars to be defined ($_ENV, $_SERVER or getenv()). It does not check the existance in a '.env' file. Throw an exception if these ENV vars are not. This is particularly useful to let people know
-any explicit required variables that your app will not work without.
+Using Dotenv, you can require specific ENV vars to be defined ($_ENV, $_SERVER or getenv()). Note: It does not check for existance of a variable in a '.env' file.
+Throw an exception if these ENV vars are not. This is particularly useful to let people know any explicit required variables that your app will not work without.
```
Comment (jpuck) [inline README.md]: This is an incomplete sentence. Might as well fix it up while you're at it.

File: README.md [hunk 2]
```diff
@@ -138,8 +138,8 @@ $dotenv->overload();
 Requiring Variables to be Set
 -----------------------------
 
-Using Dotenv, you can require specific ENV vars to be defined ($_ENV, $_SERVER or getenv()). It does not check the existance in a '.env' file. Throw an exception if these ENV vars are not. This is particularly useful to let people know
-any explicit required variables that your app will not work without.
+Using Dotenv, you can require specific ENV vars to be defined ($_ENV, $_SERVER or getenv()). Note: It does not check for existance of a variable in a '.env' file.
```
Comment (jpuck) [inline README.md]: existance is a typo, should be existence

Comment (infabo) [inline README.md]: yes.

**Comment by GrahamCampbell** [comment]
🍻

**Pull request merged**